src/main/java/net/minecraftforge/fluids/BlockFluidClassic.java

```diff
@@ -368,6 +368,17 @@ public int place(World world, BlockPos pos, @Nonnull FluidStack fluidStack, bool
         }
         if (doPlace)
         {
+            if (!world.isRemote)
```

Comment:
Yeah this is the one part I don't like.
I'm not sure what the best place to put the new method is, any ideas?